backend/src/did-resolver/did-resolver.service.spec.ts

```diff
@@ -16,10 +16,10 @@ describe('DidResolverService', () => {
                     useValue: {
                         get: jest.fn((key: string) => {
                             if (key === 'INFURA_NETWORK_ID') {
-                                return 'eee90ba565f04be7880a63ee41082f17';
+                                return process.env.INFURA_NETWORK_ID;
```

Comment:
we should hide these values. 
I changed to use jest.setFiles and call dotenv.config from inside "setupJest.js"